src/ImGui.NET/ImGuiNative.cs

```diff
@@ -738,12 +735,12 @@ internal static void igPushFont(object nativeFont)
         // public state access - if you want to share ImGui state between modules (e.g. DLL) or allocate it yourself
         [DllImport(cimguiLib)]
         public static extern string igGetVersion();
-        [DllImport(cimguiLib)]
-        public static extern void* igGetpublicState();
-        [DllImport(cimguiLib)]
-        public static extern uint igGetpublicStateSize();
-        [DllImport(cimguiLib)]
-        public static extern void igSetpublicState(void* state, bool construct);
+        /*
```

Comment:
Do we want to wrap these, or would they require more cross-cutting changes to be useful? From what I understand, this is the most interesting / useful stuff that has been added since the old versions, but I'm not sure if we'd need to add a bunch of other things to make them useful.